pages/hub/insights/index.tsx

```diff
@@ -15,10 +19,41 @@ import useStore from "lib/store";
 import useSession from "lib/hooks/useSession";
 import { useToast } from "lib/hooks/useToast";
 import Text from "components/atoms/Typography/text";
+import Card from "components/atoms/Card/card";
+import getPercent from "lib/utils/get-percent";
+import Pill from "components/atoms/Pill/pill";
+import { getRelativeDays } from "lib/utils/date-utils";
+import CardRepoList, { RepoList } from "components/molecules/CardRepoList/card-repo-list";
+import Button from "components/atoms/Button/button";
+import Icon from "components/atoms/Icon/icon";
+import Title from "components/atoms/Typography/title";
+
+const staticRepos: RepoList[] = [
```

Comment:
What are the static repos for?

Reply:
This was used to display the repos on the card... The data is static ATM, and the information on the card are all hardcoded. 
> I wonder if we create a demo account that can't log in that has some insights instead?

There is also room to improve this to live datas from @nickytonline suggestion above.

Reply:
This was used to display the repos on the card... The data is static ATM, and the information on the card are all hardcoded. 
> I wonder if we create a demo account that can't log in that has some insights instead?

There is also room to improve this to live datas from @nickytonline suggestion above.